addons/portal/static/src/scss/portal.scss

```diff
@@ -336,20 +333,11 @@ img.o_we_custom_image {
             padding-left: 0;
             padding-right: 0;
             @extend %o-portal-breadcrumbs;
+        }
 
-            .breadcrumb-item.active {
-                // In this context, active breadcrumb act as title
-                color:  $headings-color;
-                line-height: $h5-font-size/$font-size-base;
-                font: {size: $h5-font-size; weight: $headings-font-weight};
-
-                &:before {
-                    // Restore breadcrumb divider to default
-                    color: $breadcrumb-divider-color;
-                    line-height: $line-height-base;
-                    font: {size: $font-size-base; weight: $font-weight-base};
-                }
-            }
+        &.bg-white {
+            // Overwrite default navabar bg-color
+            background-color: white!important;
```

Comment:
I don't see where we put another important bg-color on that element...

Reply:
![screen shot 2018-09-19 at 10 07 32](https://user-images.githubusercontent.com/8558351/45739730-f19f5c00-bbf3-11e8-8c6a-ac0ffcd7fec6.png)

Reply:
Ok, that's because I made the mistake of customizing all navbar with the user color. Maybe what we want is only customizing the main navbar with that one, right ? I suggest to add a class on the main navbar and style only that one. That way bg utilities will work on the other navbar (like the portal one).

Note: if fact this is even a bug. Right now the portal navbar are white and when you install website, they are gray.

Reply:
@stefanorigano Let me think about this before adapting

Reply:
@stefanorigano Ok, actually, using the user color for all navbar is nice, so I suggest to keep it. This keeps your problem too unfortunately. The real question is why do you need to sometimes use white ?

Reply:
In portal we always need white navbars because of the gray page bg... 
I don't have a real solution here, rather than use the same approach we used for the body bg, so:
```scss
if ($[sidebar-bg-default] == #[default boostrab sidebar color] ) { 
    background : white!imp;
} 
```

Reply:
The point is: there is no bootstrap default color. Navbar components do not have one and even if they did, once website is installed this is the user color which is used (which we chose to be a light gray). So I think using bg-white and let website.scss override it is the solution. It will be white when website is not installed and the user color when website is installed.

But yes... I understand that with the previous body color hack this means that when website is installed and the user did not changed anything, we will have a gray portal with gray navbars. If you want to force white to solve that... maybe the condition is still the body color (and not the menu one). Maybe adding this at the top of the portal.scss file: 
```scss
$o-portal-use-default-colors: $body-bg == $white;
```
then force colors if that variable is true... but I am not sure of the real use case as the goal is that users change the colors once website is installed.

Reply:
I think it's a good solution, especially because having a white page-bg is the most common user case scenario: even when the user will customize the design, the bg color usually remain white.
Using this approach we'll ensure that the design will be ok by default and with some basic customization. 👍  thanks